### ISSUE TITLE
Add home section in the navigation bar

### ISSUE THREAD

**Issue by Revathyvenugopal162** [ansys/ansys-sphinx-theme#498]
here is the proposed solution by @mia-guo-ux 
![image](https://github.com/user-attachments/assets/27838ff4-70cf-431c-ac12-fb7b6a5d01de)

 - Add <project name>-Home in the nav section
 - Change Home icon to project name

pinging @ansys/pyansys-core  for visibility

**Comment by dipinknair** [comment]
My suggestion is to have a home icon instead of `package name + home` so that we get more space on section bar. If thats not possible just `Home` ?

**Comment by greschd** [comment]
> My suggestion is to have a home icon instead of `package name + home` so that we get more space on section bar. If thats not possible just `Home` ?

I'm of two minds about this: one the one hand, yes, we are tight on space in the title bar. On the other hand, it can then also be confusing whether `Home` refers to the PyAnsys home, or the package home.
Maybe it's worth considering what happens for packages which have a fairly long name (e.g. https://visualization-interface.tools.docs.pyansys.com/); can we come up with a short name that still conveys the "project home" meaning?

**Comment by mia-guo-ux** [comment]
> > My suggestion is to have a home icon instead of `package name + home` so that we get more space on section bar. If thats not possible just `Home` ?
> 
> I'm of two minds about this: one the one hand, yes, we are tight on space in the title bar. On the other hand, it can then also be confusing whether `Home` refers to the PyAnsys home, or the package home. Maybe it's worth considering what happens for packages which have a fairly long name (e.g. https://visualization-interface.tools.docs.pyansys.com/); can we come up with a short name that still conveys the "project home" meaning?

I agree that simply using "Home" could lead to confusion. We could standardize the name to "Package home"

**Comment by mia-guo-ux** [comment]
@Revathyvenugopal162 We can proceed with 

- A pyansys logo to go to pyansys home page 
- A package home label to corresponding home pages
